presto-orc/src/main/java/com/facebook/presto/orc/OrcReaderFeature.java

```diff
@@ -0,0 +1,37 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.orc;
+
+public enum OrcReaderFeature
+{
+    /**
+     * Enable stripe metadata cache in the reader.
+     */
+    STRIPE_META_CACHE,
```

Comment:
one suggestion is add a new field called "description" which provides info about the feature, instead of being in the comment.

Reply:
Done. + added toString()